modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/edit_kb_article.jsp

```diff
@@ -342,6 +371,7 @@ if (editKBArticleDisplayContext.isPortletTitleBasedNavigation()) {
 </aui:form>
 
 <portlet:renderURL var="scheduleModalURL" windowState="<%= LiferayWindowState.POP_UP.toString() %>">
+	<portlet:param name="resourcePrimKey" value="<%= String.valueOf(kbArticle.getResourcePrimKey()) %>" />
```

Comment:
I would pass instead both parameters we need, calling the new suggested methods of DisplayContext: `isScheduled` and `getDisplayDate`.

Note: use `String.valueOf()` for a boolean